js/index.js

```diff
@@ -1,7 +1,88 @@
-// Iteration 1: Names and Input
+// Iteration 1
 
+let hacker1 = "Alberto";
+let hacker2 = "Alberto";
 
-// Iteration 2: Conditionals
+//console.log(`Driver\'s name is: ${hacker1}`);
+//console.log(`Navigator\'s name is: ${hacker2}`);
 
+// Iteration 2
 
-// Iteration 3: Loops
\ No newline at end of file
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.lenght} characters`
+  );
+} else if (hacker2.length > hacker1.length) {
+  console.log(
+    `It seems that the navigator has the longest name, it has ${hacker2.length} characters`
+  );
+} else {
+  console.log(
+    `Wow, you both have equally long names, ${hacker1.length} characters!`
+  );
+}
+
+// Iteration 3
+
+//creamos un loop por cada carácter en el nombre de hacker1
+let upperHack = "";
+
+for (let i = 0; i < hacker1.length; i++) {
+  upperHack += hacker1[i].toUpperCase() + " ";
+}
+console.log(upperHack);
```

Comment:
Very good! Now if you print the upperHack between quotes, you'll see that you get a space at the end of the string => "A L B E R T O "
You could handle this case by checking if the character that you're iterating over is the last character of that string, and in that only case, not add any space at the end (cf. Uros' review)
Other solution (that do not involve loops): you could use the trim() method, that removes whitespace from both ends of a string